docs/guides/1239-project-file-organization.md

```diff
@@ -0,0 +1,215 @@
+---
+slug: /1239/project-file-organization
+displayed_sidebar: "0.2"
+---
+
+# Project file organization
+
+As time will pass, your actions will get bigger and bigger. You will feel the need to better organize your file structure by moving some parts in subfolders.
+
+The easiest way, when iterating fast, is to create a global module.
```

Comment:
I think "root module" is better than "global module", but that's always already created. The difference is that the default is an **anonymous** module (which means `module: ""`), and what we need to change is give a name to that module.